driver/driver.go

```diff
@@ -143,15 +146,14 @@ func NewDriver(ep, token, url, region, doTag, driverName, debugAddr string) (*Dr
 		name:                  driverName,
 		publishInfoVolumeName: driverName + "/volume-name",
 
-		doTag:     doTag,
-		endpoint:  ep,
-		debugAddr: debugAddr,
-		hostID:    func() string { return hostID },
-		region:    region,
-		mounter:   newMounter(log),
-		log:       log,
-		// we're assuming only the controller has a non-empty token.
```

Comment:
The comment was moved to the variable assignment for `isController`. Since it's a variable I think it's no longer relevant on this line, but only earlier where we actually make that assumption.

Reply:
Ah sorry, totally missed the move. Makes complete sense. 👍